scio/src/main/scala/org/apache/zeppelin/scio/DisplayTapImplicits.scala

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.scio
+
+import com.google.api.services.bigquery.model.TableSchema
+import com.spotify.scio.bigquery.TableRow
+import com.spotify.scio.io.Tap
+import com.spotify.scio._
+import org.apache.avro.Schema
+import org.apache.avro.generic.GenericRecord
+
+import scala.concurrent.Future
+import scala.reflect.ClassTag
+
+/**
+ * Implicit Zeppelin display helpers for [[Tap]] and [[Future]] of a [[Tap]].
+ */
+object DisplayTapImplicits {
```

Comment:
Same answer as above. Would prefer to keep it this way, same reasons:
 * difference in display methods
 * safer
 * not big overhead